Formula/d/dadroit_json_generator.rb

```diff
@@ -0,0 +1,41 @@
+class DadroitJsonGenerator < Formula
+  desc "High-performance tool for generating nested JSON data using custom templates"
+  homepage "https://github.com/DadroitOrganization/Generator"
+  url "https://github.com/DadroitOrganization/Generator/archive/refs/tags/Release_Version_1.0.0.352.tar.gz"
+  sha256 "2ef4e55bb00d5a61dad40bc050de7b8d56611e777f9db10c3e9997d5a2ade156"
+  license "GPL-3.0-or-later"
+  # Dependency on Free Pascal Compiler
+  depends_on "fpc" => :build
+
+  def install
+    # Cloning the mORMot2 repository, required for building the tool
+    system "git", "clone", "https://github.com/synopse/mORMot2.git", "mORMot2"
```

Comment:
This should be it's own formula

Reply:
Would it be permissible to include the open-source 'mormot2' package directly within our project's source code, creating a combined repository for a new release on our GitHub project? This approach would centralize all necessary source code, eliminating the need for any external 'git clone' operations beyond our Generator code base. Is this method acceptable for managing dependencies in GitHub projects? in homebrew core pull requests?